tracer/test/test-applications/security/Samples.Security.AspNetCore5/rasp-rule-set.json

```diff
@@ -143,6 +144,50 @@
       "on_match": [
         "customblock"
       ]
+    },
```

Comment:
Not something necessary a change for this PR, but why do we need external rules? Seems these have been added to the main rule set already (plus the file is added to several samples)

Reply:
This file is used when testing manually AspNetCore5. I was thinking about merging the three files into just one, used by the unit tests, integration and this sample.

Reply:
I have removed one of them. Right now, we are using external rules for these tests, but now that we have the main ruleset, we could rely on them. I could write a separate PR for it. Thanks!